domain/domain.go

```diff
@@ -1472,7 +1472,7 @@ func (do *Domain) distTaskFrameworkLoop(ctx context.Context, taskManager *storag
 			return
 		}
 		var err error
-		dispatcherManager, err = dispatcher.NewManager(ctx, taskManager)
+		dispatcherManager, err = dispatcher.NewManager(ctx, taskManager, schedulerManager.GetID())
```

Comment:
We can pass serverID to distTaskFrameworkLoop. Don't use id from schedulerManager since this will coupling schedulerManager and dispatcherManager